agent/prototype2_agentCore.py

```diff
@@ -14,129 +14,147 @@
 from agent.VPSBuyer import VPSBuyer
 from agent.Wallet import Wallet
 
-
+try:
+    from rpc.client import Client
+    import thread
+except Exception as e:
+    print("Importing of RPC modules failed because of %s, now running without rpc." % str(e))
 
 from ssh.SSH import SSH
 
 from Tribler.community.tunnel.tunnel_community import TunnelSettings
+        
+def update_rpc_server():
```

Comment:
maybe add a comment about what the function is supposed to do 

i.e.

"""
This function was written to send logging about the agent to a central server

"""